MAUI/TextInputLayout/Assistive-Labels.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: Assistive Labels in .NET MAUI Text Input Layout control | Syncfusion
+description: Learn here all about Assistive Labels support in Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Assistive Labels in .NET MAUI Text Input Layout (SfTextInputLayout)
+
+Assistive labels provide additional information about text entered in the input view controls.
+
+## Helper text
+
+Helper text conveys additional guidance about the input field such as how it will be used. It can be set using the `HelperText` property.
+
+{% tabs %} 
+
+{% highlight xaml %} 
+
+<inputLayout:SfTextInputLayout Hint="Name"
+                               HelperText="Enter your name">
+    <Entry />
+</inputLayout:SfTextInputLayout>   
+
+{% endhighlight %}
+
+{% highlight C# %} 
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Name";
+inputLayout.HelperText = "Enter your name";
+inputLayout.Content = new Entry(); 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+The visibility of the helper text can be disabled by setting the `ShowHelperText` property to false. By default, it is set to true.
+
+![helper label](images/AssistiveLabels/HelperText.jpg)
+
+## Error message
+
+When the text input is not accepted, an error message will display instructions to fix it. Error messages will be displayed below the input line till entering the correct text. It can be set using the `ErrorText` property, but it will be displayed only when the `HasError` property is set to `true`.
```

Comment:
When the text input is not accepted, an error message will display instructions to fix it. Error messages will be displayed below the input line till entering the correct text. It can be set using the `ErrorText` property, but it will be displayed only when the `HasError` property is set to `true.`

Reply:
Addressed the changes